src/talk/renderer/Viewer/ViewerHandlerBase.vue

```diff
@@ -38,7 +38,7 @@ const readyToShow = computed(() => !props.loading && !props.error)
 	<div class="viewer-wrapper">
 		<NcEmptyContent
 			v-if="loading"
-			:name="t('talk_desktop', 'Loading …')"
+			:name="t('talk_desktop', 'Loading …')"
```

Comment:
So, we're touching the strings?

Reply:
@susnux Do you know if this auto fix also keeps translations in Transifex for apps, or it only works in libs? 

If it doesn't, we probably should remove auto-fixing in this rule by default, as it may break a lot of translations...

Reply:
No it will only use the fuzzy translation finder meaning it will be marked as not reviewed and needs a new review.
But this is expected.

Also those fixes are expected - using a plain space here is the same as any other typo as per our localization guide. So as long as its not string freeze it should be ok to do so and fix this.

Reply:
The only rule I know is this one: https://github.com/nextcloud/server/wiki/Translation-knowledge-(valid-for-whole-nextcloud-project)/d45f1d248fdc6b1509337ebb0017653649a8f440#ellipsis
- It asks to use space in a sentence and no space when shortening a word
  - Example: 'Uploading …', same as Loading - is a full word.
- It doesn't ask to use non-breaking space...

Reply:
But typographical there must be a non breaking space as otherwise the word wrap would be wrong.
Similar like between numbers and units etc